docs/examples/hello_cross_val.rst

```diff
@@ -110,36 +110,14 @@ and adding a ``random_epsilon`` before returning the results packaged with a DXO
   NVIDIA FLARE can be used with any data packaged inside a :ref:`Shareable <shareable>` object (subclasses ``dict``), and
   :ref:`DXO <data_exchange_object>` is recommended as a way to manage that data in a standard way.
 
-Application Configuration
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Inside the config folder there are two files, ``config_fed_client.json`` and ``config_fed_server.json``.
-
-.. literalinclude:: ../../examples/hello-world/hello-numpy-cross-val/jobs/hello-numpy-cross-val/app/config/config_fed_server.json
-   :language: json
-   :linenos:
-   :caption: config_fed_server.json
-
-The server now has a second workflow configured after Scatter and Gather, :class:`CrossSiteModelEval<nvflare.app_common.workflows.cross_site_model_eval.CrossSiteModelEval>`.
-
-The components "model_locator" and "formatter" have been added to work with the cross site model evaluation workflow,
-and the rest is the same as in :doc:`Hello Scatter and Gather <hello_scatter_and_gather>`.
-
-
-.. literalinclude:: ../../examples/hello-world/hello-numpy-cross-val/jobs/hello-numpy-cross-val/app/config/config_fed_client.json
-   :language: json
-   :linenos:
-   :caption: config_fed_client.json
-
-The client configuration now has more tasks and an additional Executor ``NPValidator`` configured to handle the "validate" task.
-The "submit_model" task has been added to the ``NPTrainer`` Executor to work with the :class:`CrossSiteModelEval<nvflare.app_common.workflows.cross_site_model_eval.CrossSiteModelEval>`
-workflow to get the client models.
-
 Cross site validation!
 ----------------------
 
-.. |ExampleApp| replace:: hello-numpy-cross-val
-.. include:: run_fl_system.rst
+We can run it using NVFlare simulator
+
+```bash
```

Comment:
Minor thing, this syntax only works for markdown, for rst we can instead use `.. code-block:: bash `